helper-cli/src/main/kotlin/commands/ListPackagesCommand.kt

```diff
@@ -59,12 +68,11 @@ class ListPackagesCommand : CliktCommand(
                 .filter(LicenseView.ONLY_DETECTED)
                 .map { it.license.toString() }
 
-        val packages = ortResult
-            .collectProjectsAndPackages()
-            .filter {
+        val packages = ortResult.collectProjectsAndPackages().filter { id ->
+                (ortResult.isPackage(id) && PACKAGES in type) || (ortResult.isProject(id) && PROJECTS in type)
```

Comment:
One more nit: For performance reasons, you should do the cheap `in type` checks first before the more expensive `isPackage()` / `isProject()` checks.

Reply:
Hm, I believe performance is really not relevant here.

Reply:
I believe it'd be good practice anyway; if you don't always do it, it's easy to forget in cases where it really matters. But your call, I'll approve anyway.

Reply:
I believe it's also valuable to code review practices to not discuss irrelevant things to keep things focused. IMO this should be weighed against each other. Anyhow, readability-wise the order IMO does not matter, so I'll just merge it as is.